subprojects/ide/src/integTest/groovy/org/gradle/plugins/ide/eclipse/EclipseProjectFixture.groovy

```diff
@@ -40,10 +46,24 @@ class EclipseProjectFixture {
         return this.project.'comment'.text()
     }
 
+    Node getFilteredResourcesNode() {
+        def r = this.project.filteredResources
+        return r ? r[0] : null
+    }
+
     void assertHasReferencedProjects(String... referencedProjects) {
         assert this.project.projects.project*.text() == referencedProjects as List
     }
 
+    void assertHasResourceFilterXml(String expectedFilteredResourcesXml) {
```

Comment:
Now that the ids are sequential and predictable, most of this code is now unnecessary as we can just put the expected id in the xml.  I will remove it.